tailwind.config.js

```diff
@@ -5,7 +5,11 @@ module.exports = {
   },
   purge: [],
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        mono: ['IBM Plex Mono', 'monospace'],
```

Comment:
Maybe use this approach: https://github.com/tailwindlabs/tailwindcss/discussions/2060